templates/new_service/.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Deploy
+concurrency: build_and_deploy_${{ github.ref_name }}
+
+on:
+  push:
+   branches:
+    - main
+  pull_request:
+    branches:
+    - main
+    types: [opened, reopened, synchronize, labeled]
+
+permissions:
+  contents: write
+  deployments: write
+  packages: write
+  pull-requests: write
+
+env:
+  TERRAFORM_BASE: terraform/application
+  HEALTHCHECK_CMD: 'healthcheck/all'
+
+jobs:
+  build:
+    name: Build
+    env:
+      DOCKER_IMAGE: #DOCKER_REPOSITORY#
```

Comment:
DOCKER_IMAGE usually includes the tag. Rename to DOCKER_REPOSITORY ?

Reply:
changed